packages/mdc-chips/chip-set/test/foundation.test.ts

```diff
@@ -195,7 +179,7 @@ describe('MDCChipSetFoundation', () => {
      () => {
        const {foundation} = setupTest();
 
-       foundation['selectedChipIds_'] = [];
```

Comment:
There might be a reason to use indexing over dot notation. My guess is that for IE 11 compatibility.

is this change required for this PR?

Reply:
I'm not sure why that's there. I believe someone else's changes got merged in here

Reply:
Can you please revert these changes if not related to this PR?